demisto_sdk/commands/common/constants.py

```diff
@@ -817,7 +817,7 @@ class GithubContentConfig:
     def __init__(self, repo_name: Optional[str] = None):
         if not repo_name:
             try:
-                urls = GitUtil().repo.remote().urls
+                urls = list(GitUtil().repo.remote().urls)
```

Comment:
doesn't `.urls` return a list already?

Reply:
a generator. 
So when you're trying to iterate over it again it fails.